syft/pkg/license_test.go

```diff
@@ -97,3 +95,134 @@ func Test_Sort(t *testing.T) {
 
 	}
 }
+
+func TestLicense_Merge(t *testing.T) {
```

Comment:
So merge only ever merges the URLs between two licenses objects, and requires all other fields to be equal? Just checking my understanding, because `Merge` seems like the name of a more general function than that.

Reply:
it also merges location sets on licenses.

> Merge seems like the name of a more general function than that.

I'm not certain what you mean, are you recommending a rename of the name of the function `Merge`?

Reply:
It's exported, so I don't think we should bother renaming it. I was just remarking that the current name seems surprising to me, but maybe I don't understand the semantics of the license package well. I don't think any change is needed here.